src/main/resources/view/MainWindow.fxml

```diff
@@ -19,8 +19,8 @@
   <scene>
     <Scene>
       <stylesheets>
-        <URL value="@DarkTheme.css" />
         <URL value="@Extensions.css" />
+        <URL value="@DarkTheme.css" />
```

Comment:
Is there a reason for this change?

Reply:
I'm forcing the theme file to be the right-most element in the array returned by getStyleSheets(). This is so that I can hot-swap it by modifying array[1].

If this is not changed, the first instance of theme changing will require me to delete array[0] and then append the light theme. Subsequent theme changes will require deleting array[1], which is not very nice to do

Reply:
I see, relying on position of stylesheet doesn't sound like a good idea. Any chance you can store the theme stylesheet position somewhere, and change the value programatically?

Reply:
Would it be better to wipe the stylesheets and then re-add the relevant styles on each request? Here is the example:
```
public class MainWindow extends UiPart<Stage> {
    @Subscribe
    private void handleChangeThemeEvent(ChangeThemeRequestEvent event) {
        final String extensions = "view/Extensions.css";
        primaryStage.getScene().getStylesheets().clear();
        primaryStage.getScene().getStylesheets().add(event.theme);
        primaryStage.getScene().getStylesheets().add(extensions);
    }
}
```

Reply:
I'm not sure what the user will see when there are no stylesheets (however short). I think the current approach is okay, but it limits the possibilities wrt to stylesheets, but so does the approach above

Reply:
Alright I just checked that the ObservableList object (returned by getStyleSheets) has a method called setAll(String... elements) that clears the list and then adds the vararg elements in. The only difference is that the method only sends out one single change event to listeners, so there wont be any "down time" when there are no stylesheets. This would be the method:
```
public class MainWindow extends UiPart<Stage> {
    @Subscribe
    private void handleChangeThemeEvent(ChangeThemeRequestEvent event) {
        final String extensions = "view/Extensions.css";
        primaryStage.getScene().getStylesheets().setAll(extensions, event.theme);
    }
}
```
I think this would also eliminate the dependency on the order with which styles are loaded in the fxml. With this, I can revert the order of styles in MainWindow.fxml

Of course, we can also go with the original implementation and I'll rename the magic index number for clarity. Do let me know which one feels better, and I'll do up the changes in a new commit.

Reply:
How about this:

https://docs.oracle.com/javase/8/javafx/api/javafx/collections/ObservableList.html

There seems to be a `removeAll()` method. We can store the current theme as a property, and:

```
primaryStage.getScene().getStylesheets().removeAll(currentTheme);
primaryStage.getScene().getStylesheets().add(newTheme);
this.currentTheme = newTheme;
```

Reply:
Roger. I've modified the existing implementation in a new commit that has been pushed to the PR. On top of storing the current theme, I now load the default dark theme on initialization programatically. As such, my commit also removes the CSS element in MainWindow.fxml because it's not needed anymore.